google.json

```diff
@@ -1,13 +1,200 @@
+        "label": "Accessibility sound amplifier",
+        "description": "Accessibility sound amplifier",
+        "web": [
+            "https://play.google.com/store/apps/details?id=com.google.android.accessibility.soundamplifier"
+        ],
+        "removal": "delete"
+    },
+    {
+        "id": "com.google.android.gms.supervision",
+        "label": "Family Link parental controls",
+        "description": "It has something to Family Link parental controls.\nIntroduced in android 13.",
+        "web": [
+            "https://play.google.com/store/apps/details?id=com.google.android.gms.supervision"
+        ],
+        "removal": "delete"
+    },
+    {
+        "id": "com.google.android.overlay.gmsconfig.searchselector",
+        "description": "Not needed for (com.google.android.apps.setupwizard.searchselector).",
+        "removal": "delete"
+    },
+    {
+        "id": "com.google.android.overlay.gmsconfig.personalsafety",
+        "description": "Not needed for (com.google.android.apps.safetyhub).",
+        "removal": "delete"
+    },
+    {
+        "id": "com.google.android.ondevicepersonalization.services",
+        "description": "OnDevicePersonalization. Another thing to AdServices privacy sandbox.\nIntroduced in Android 13.",
+        "web": [
+            "https://source.android.com/docs/core/ota/modular-system/ondevicepersonalization"
+        ],
+        "removal": "delete"
+    },
+    {
+        "id": "com.google.mainline.adservices",
+        "description": "Adservices Train Version Package. AdServices too.\nuses ondevicepersonalization.\nIntroduced in Android 13\nNOTE: This package is a mandatory mainline module, which is also not documented. I don't trust it when it comes to adservices.",
+        "removal": "replace"
+    },
+    {
+        "id": "com.google.android.federatedcompute",
+        "description": "Another component of OnDevicePersonalization. But this app learns things about users.\nIntroduced in Android 13.",
+        "web": [
+            "https://source.android.com/docs/core/ota/modular-system/ondevicepersonalization"
+        ],
+        "removal": "delete"
+    },
+    {
+        "id": "com.google.android.overlay.gmsconfig.photos",
+        "description": "overlay to\nconfig_systemGallery to com.google.android.apps.photos in motorola. Useless.",
+        "removal": "delete"
+    },
+    {
+        "id": "com.google.android.overlay.gmsconfig.searchlauncherqs",
+        "description": "Useless gmsconfig. Its config default launcher but not needed.",
+        "removal": "delete"
+    },
     {
         "id": "com.android.chrome",
         "label": "Google Chrome",
-        "description": "Google Chrome: Slow & Painful\nOccasionally runs in the background, not to mention how it tracks everything.",
+        "description": "Google Chrome: Slow & Painful\nOccasionally runs in the background, not to mention how it tracks everything.\nI would say you should try brave or iceraven.",
```

Comment:
We don't suggest stuffs in the description. There's a dedicated section for it (which is already filled).